server/util/cosmos.js

```diff
@@ -220,15 +116,10 @@ export async function sendCoins(signer, targets) {
   } while (retry && retryCount < RETRY_LIMIT);
   try {
     while (!txHash) {
-      const { sequence: seq } = await getAccountInfo(signer.cosmosAddress);
-      pendingCount = Number(seq);
-      signedTx = signTransaction({
-        signer,
-        accNum,
-        stdTx,
-        sequence: pendingCount,
-      });
-      txHash = await sendTransaction(signedTx);
+      const { transactionHash } = await signingClient.signAndBroadcast(
+        delegatorAddress, msg, fee, memo,
+      );
+      txHash = transactionHash;
```

Comment:
Please use `pendingCount` as `sequence` here, we want to be able to queue up multiple transactions at a time with correct sequence